config/initializers/highlight_override.rb

```diff
@@ -32,9 +34,7 @@ def list_invalid_properties
       return invalid_properties
     end
 
-    if location_strategies.any?(&:nil?)
-      invalid_properties.push('invalid strategy detected')
-    end
+    invalid_properties.push('invalid strategy detected') if location_strategies.any?(&:nil?)
```

Comment:
more specific error like "no location strategies provided"?

actually note that this check is already done in the generated `NewHighlight` binding, so leaving this would leave a duplicate error.